src/runtime/create.ts

```diff
@@ -132,21 +132,20 @@ export function setupRuntime(notebook: StarboardNotebookElement): Runtime {
           break; // IDs should be unique, so after we find it we can stop searching.
         }
       }
-
       if (insertNewCell) {
         controls.insertCell({}, "after", id);
-        cellElements = rt.dom.cells;
       }
       if (focus === "previous") {
         window.setTimeout(() => {
-          const next = cellElements[idxOfCell - 1];
-          if (next) next.focusEditor();
+          const next = rt.dom.cells[idxOfCell - 1];
+          if (next) next.focusEditor({ position: "end" });
         });
       } else if (focus === "next") {
         window.setTimeout(() => {
-          const next = cellElements[idxOfCell + 1];
+          const next = rt.dom.cells[idxOfCell + 1];
           if (next) {
-            next.focusEditor();
+            console.log("focusing next cell");
```

Comment:
Remove debug print

Reply:
Oh thanks for catching that, totally missed it when committing the files